dev/2017/spring/breaking-down-debt/breaking-down-debt.html

```diff
@@ -0,0 +1,216 @@
+                                <p class="cite">&mdash; <strong>Bryce Nelson</strong>, a first-generation UCF marketing student who has utilized Student Financial Assistance counseling, financial aid and work-study opportunities to earn an education with minimal debt.</p>
+                            </div>
+                            <img src="[image filename='student']" alt="" class="student">
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="container">
+        <div class="row">
+            <div class="col-md-10 col-sm-10 col-md-offset-1 col-sm-offset-1">
+                <h2>Debt In Context</h2>
+                <p class="chart-description">Where average debt is concerned, the type of institution a student attends matters.</p>
+                <hr>
+                <canvas id="debt-in-context" width="400" height="200"></canvas>
+
+                <p class="chart-footer">* 2012 data for for-profit, private, and public. 2014-15 data for Florida and 2016 for UCF.</p>
+                <p class="chart-alt-footer">Association of Public &amp; Land-Grant Universities, 2016; The Institute for College Access &amp; Success, 2014; UCF Office of Student Financial Assistance, 2016</p>
+
+                <h2>Brighter Futures</h2>
+                <p class="chart-description">Nearly half of UCF undergraduates carry no debt when they graduate.</p>
+                <hr>
+                <div class="chart-legend text-center">
+                    <span class="legend-container width-auto"><span class="for-profit-legend"></span> For-Profit</span> <span class="legend-container width-auto"><span class="private-non-profit-legend legend-margin-left"></span> Private Non-Profit</span> <span class="legend-container width-auto"><span class="public-legend"></span> Public</span> <span class="legend-container width-auto"><span class="florida-legend legend-margin-left"></span> National</span> <span class="legend-container width-auto"><span class="ucf-legend legend-margin-left"></span> UCF *&dagger;</span>
+                </div>
+
+                <p class="text-center"><img src="[image filename='brighter-futures']" alt="" class="brighter-futures-chart"></p>
+
+                <p class="chart-footer">* 2012 data for for-profit, private and public. 2014-15 data for SUS and UCF.<br>&dagger; FTIC students only</p>
+                <p class="chart-alt-footer">Association of Public &amp; Land-Grant Universities, 2016; The Institute for College Access &amp; Success, 2014; Florida Board of Governors System Survey of University Work Plans, 2016</p>
+            </div>
+        </div>
+    </div>
+    <div class="container-wide callout-outer">
+        <div class="callout">
+            <div class="container">
+                <div class="row content-wrap">
+                    <div class="col-md-12 col-sm-10 col-sm-offset-1 callout-inner text-left dale-container">
```

Comment:
Add a .col-md-offset-0 to this div (line 86) to unset the -sm breakpoint's offset and prevent horizontal scrollbars at -md+